### PR TITLE
Add Gremlin Section to Programming Books

### PR DESCRIPTION
**Pull request by joaoluizn**
## What does this PR do?
Add resource(s)

## For resources
Free Book

### Description
Gremlin Section and Book from Kevin Lawrence free and open source

### Why is this valuable (or not)?
There is yet not reference about gremlin query language for graph db

### How do we know it's really free?
Its a open source book (https://github.com/krlawrence/graph)

### For book lists, is it a book? For course lists, is it a course? etc.
Its a Book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -849,6 +850,11 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Learning graphqL](https://riptutorial.com/Download/graphql.pdf) - Compiled from StackOverflow Documentation (PDF)
 
 
+### Gremlin
+
+* [Practical Gremlin - An Apache TinkerPop Tutorial](https://www.kelvinlawrence.net/book/PracticalGremlin.pdf) - Kelvin R. Lawrence (PDF) - [HTML Version](https://www.kelvinlawrence.net/book/PracticalGremlin.html)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:855]: only one link, please. choose the target that links to the other.
```suggestion
* [Practical Gremlin - An Apache TinkerPop Tutorial](https://www.kelvinlawrence.net/book/PracticalGremlin.html) - Kelvin R. Lawrence
```
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. Is Gremlin a GraphQL variant? Could the two be combined under "Graph Query Languages"? Are there other graph query languages?

**Comment by joaoluizn** [comment]
Im afraid they are not related (GraphQL and Gremlin) GraphQL is a query language for specific API responses while Gremlin is a query language used on Graph databases based on Apache Tinkerpop. 

AWS Neptune and Cosmos DB Gremlin uses it.

Its more related to graph database itself. It could be related with Neo4J as a Graph database

**Comment by joaoluizn** [comment]
I can create a section called Graph Databases, where Neo4j and Apache Tinkerpop (Gremlin language) lays inside it. what do you think @eshellman

**Comment by eshellman** [comment]
I like that! Thanks.

**Pull request merged**